lms/djangoapps/branding/api.py

```diff
@@ -0,0 +1,214 @@
+        ).format(
+            link_start=u"<a href='https://www.edx.org/'>",
+            link_end=u"</a>"
+        )
+
+    return data
+
+
+def heading(is_edx_domain):
+    """ Returns the heading text copy
+    """
+    if is_edx_domain:
+        data = _(
+            "{EdX} offers interactive online classes and MOOCs from the world's best universities. "
+            "Online courses from {MITx}, {HarvardX}, {BerkeleyX}, {UTx} and many other universities. "
+            "Topics include biology, business, chemistry, computer science, economics, finance, "
+            "electronics, engineering, food and nutrition, history, humanities, law, literature, "
+            "math, medicine, music, philosophy, physics, science, statistics and more. {EdX} is a "
+            "non-profit online initiative created by founding partners {Harvard} and {MIT}."
+        ).format(
+            EdX="EdX", Harvard="Harvard", MIT="MIT", HarvardX="HarvardX", MITx="MITx",
+            BerkeleyX="BerkeleyX", UTx="UTx"
+        )
+    else:
+        data = ""
+    return data
+
+
+def social_links():
+    """ Returns the list of social link of footer
+    """
+    links = []
+    for social_name in settings.SOCIAL_MEDIA_FOOTER_NAMES:
+        links.append(
+            {
+                "provider": social_name,
+                "title": unicode(settings.SOCIAL_MEDIA_FOOTER_DISPLAY.get(social_name, {}).get("title", "")),
+                "url": settings.SOCIAL_MEDIA_FOOTER_URLS.get(social_name, "#"),
+                "social_icon": settings.SOCIAL_MEDIA_FOOTER_DISPLAY.get(social_name).get("icon", "")
+            }
+        )
+    return links
+
+
+def about_edx_link():
+    """ Returns the list of marketing links of footer
+    """
+
+    return [
+        {
+            "title": _("About"),
+            "url": marketing_link('ABOUT')
+        },
+        {
+            "title": _("News"),
+            "url": marketing_link('NEWS')
+        },
+        {
+            "title": _("Contact"),
+            "url": marketing_link('CONTACT')
+        },
+        {
+            "title": _("FAQ"),
+            "url": marketing_link('FAQ')
+        },
+        {
+            "title": _("edX Blog"),
+            "url": marketing_link('BLOG')
+        },
+        {
+            "title": _("Donate to edX"),
+            "url": marketing_link('DONATE')
+        },
+        {
+            "title": _("Jobs at edX"),
+            "url": marketing_link('JOBS')
+        }
+    ]
+
+
+def get_footer_logo(site_name, is_edx_domain):
+    """ Return the logo used for footer about link
+
+    Args:
+        site_name(str): The site url to get the absolute link
+        is_edx_domain(bool): Flag to check is it in edx domain or open edx
+
+    Returns:
+        Absolute url to logo
+    """
+    if is_edx_domain:
+        logo_file = 'images/edx-theme/edx-header-logo.png'
+    else:
+        logo_file = 'images/default-theme/logo.png'
+    try:
```

Comment:
For readability, please separate this exception handler from the if/else block above it with a newline.